content/learn/book/welcome/plugins/_index.md

```diff
@@ -5,12 +5,76 @@ template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: Explain how plugins work
+Writing all your Bevy code in one file would be a pain. Even a game as simple as snake could take up more than 300 lines of code. That's what plugins are for.
+
+Plugins are essentially collections of systems, components and resources that can be added to the main app. For example, if you want a window to display, you can add the `WindowPlugin` and the `RenderPlugin`.
+
+You can also define your own plugins to seperate related pieces of information into dedicated areas. Finally, there are also plugin groups that group plugins together. 
 
 ## `MinimalPlugins`
 
+One of the aformentioned plugin groups is the `MinimalPlugins` group. This plugin is for the most bare-bones of games or tools. It only includes bevy-core and bevy-app. If you are making a text-based game, this is the option for you.
```

Comment:
Adding links to the documentation (on docs.rs usually) helps curious readers dig into the details of the topic without distracting from the main flow.